code/procs/scorestats.dm

```diff
@@ -86,6 +87,14 @@ var/datum/score_tracker/score_tracker
 
 		final_score_sec = (score_crew_survival_rate + score_enemy_failure_rate) * 0.5
 
+		for(var/mob/living/carbon/human/npc/monkey/stirstir/M in mobs)
+			if(isalive(M))
+				score_stirstir_alive = TRUE
```

Comment:
Would suggest flipping to "if dead then FALSE", cause as it is sec will get penalised if a map doesn't spawn StirStir on it. IDK if any still don't but that wasn't always guaranteed.

(although, then you could just gib StirStir's body and avoid the penalty, so maybe `score_stirstir_alive` ought to be global & set by StirStir's death proc directly.)

Reply:
If it's set by death then it'd need to be re-set by all revive methods and that becomes a mess quickly. I think a map not spawning with stirstir somewhere is unusual enough to be considered an error state.

Reply:
stir stir spawns on every map in rotation and even some that aren't I made sure to check